dashboard/app/models/pd/workshop_survey.rb

```diff
@@ -347,7 +368,80 @@ def self.options
         "7",
         "8",
         "9",
-        "10+",
+        "10+"
+      ],
+
+      hours_per_week: [
+        "0 to 30 minutes",
+        "Between 30 minutes and an hour",
+        "Over an hour (up to 90 minutes)",
+        "90 minutes to 3 hours",
+        "4 to 5 hours",
+        "More than 5 hours"
+      ],
+
+      weeks_per_year: [
+        "Less Than a Quarter( 8 Weeks or Less)",
```

Comment:
nit: misplaced whitespace (looks like the same misplace is in the source doc you were given, but it would still be nice to clean it up)